fastlane/lib/fastlane/actions/hockey.rb

```diff
@@ -187,6 +186,13 @@ def self.run(options)
         end
       end
 
+      def self.puts_download_url(body)
```

Comment:
Not sure how picky one should be but now the method name is not doing what it says :) 

Good update otherwise 👍